Extending/Content-Apps/index.md

```diff
@@ -167,6 +170,7 @@ Here is an example where all types are taken intro consideration when limiting a
                 "+member/*", // show for all other Member types
                 "-contentType/textPage", // hide on Content Type with alias 'textPage'
                 "+contentType/*", // show for all other Content types
+                "-contentType/*", // hide for all other Content types
```

Comment:
When adding the `"-contentType/*", // hide for all other Content types`, I think it makes sense to remove the two other lines referencing the contentType 🤔 
What do you think?

Reply:
@sofietoft yes, I think it makes sense to remove the **_"+contentType/*", // show for all other Content types_** since we are mentioning hide. 

I think we should leave the alias one  **_"-contentType/textPage", // hide on Content Type with alias 'textPage'_** since it acts as an alias example and it will be consistent with the first two lines

Let me know your thoughts and I'll make the changes accordingly :)

Reply:
Right! That's a good point 🤔 I think keeping the specific one for the textPage is a good idea!

Let's just get rid of `"+contentType/*"` 💪

Reply:
@sofietoft Done.